cli/cli_click.go

```diff
@@ -2,14 +2,31 @@ package cli
 
 import (
 	"bufio"
+	"context"
 	"fmt"
+	"net/url"
 	"os"
 
+	"github.com/NordSecurity/nordvpn-linux/daemon/pb"
 	"github.com/urfave/cli/v2"
 )
 
 // Click is hidden cmd used to open some info when clicking on desktop icon
 func (c *cmd) Click(ctx *cli.Context) error {
+	if ctx.NArg() == 1 {
+		url, err := url.Parse(ctx.Args().First())
+		if err != nil {
+			return formatError(err)
+		}
+
+		if url.Scheme == "nordvpn-sl" && url.Host == "claim-online-purchase" {
+			_, err := c.client.ClaimOnlinePurchase(context.Background(), &pb.Empty{})
+			if err != nil {
+				return formatError(err)
+			}
+		}
+	}
+
 	if ctx.NArg() >= 1 {
```

Comment:
this code will always execute also for `if url.Scheme == "nordvpn-sl" && url.Host == "claim-online-purchase" {` and will return the error `Expected a url with nordvpn scheme.`